src/main/java/com/dougnoel/sentinel/steps/APISteps.java

```diff
@@ -150,12 +150,19 @@ public static void verifyResponseTime(double time) {
 
 	/**
 	 * Validates text in an API response.
-	 * 
+	 * <p>
+	 * <b>Gherkin Examples:</b>
+	 * <ul>
+	 * <li>I validate the response <b>contains</b> the text <b>"key":"value"</b></li>
+	 * <li>I validate the response <b>does not contain</b> the text <b>"value":"key"</b></li>
+	 * <li>I validate the response <b>has</b> the text <b>123456789ABCDEFG</b></li>
+	 * </ul>
+	 * <p>
 	 * @param assertion String null to see if the text exists, "does not" to see if it is absent
 	 * @param matchType String use "contains" for a partial match otherwise it will be an exact match
 	 * @param text String the text to match
 	 */
-	@Then("^I validate the response( does not)? (has|have|contains?) the text \"([^\"]*)\"$")
+	@Then("^I validate the response( does not)? (has|have|contains?) the text (.*?)$")
```

Comment:
The way to solve this si to write our own contains and equals evaluation methods in SentinelStringUtils that checks the conditions with surrounding quotes, and then without before returning a result so that we don't fail based on someones assumptions and so we don't break old code.

Reply:
I know we had similar discussion for text verification but it looks like we never implemented those changes (and probably should)?
TextVerificationSteps line 74: `@Then("^I verify the (.*?)( does not)? (has|have|contains?) the text \"([^\"]*)\"$")`
Last I checked we wanted to drop quote checking to free the user to put in exactly what they wanted, lest we run slower to accommodate them. I agree that we don't want to break old code but we're rapidly approaching a 2.0 if we're slowing down our code like this.